### PR TITLE
experimental: delete docker images after runs

### PR DESCRIPTION
**Pull request by DavidKorczynski**
otherwise the storage is getting eaten too fast

### CODE REVIEW

File: experimental/c-cpp/runner.py [hunk 1]
```diff
@@ -251,6 +251,15 @@ def run_on_targets(target,
               build_heuristics=build_heuristics,
               generator_heuristics=generator_heuristics)
 
+  # Cleanup the OSS-Fuzz docker image
+  clean_up_cmd = [
+      'docker', 'image', 'rm', f'gcr.io/oss-fuzz/{worker_project_name}'
+  ]
+  try:
+    subprocess.check_call(' '.join(clean_up_cmd), shell=True)
```
Comment (DonggeLiu) [inline experimental/c-cpp/runner.py:259]: No need to join, IIRC?

**Review by DonggeLiu — APPROVED** [review]

File: experimental/c-cpp/runner.py (hunk 1 above)
Comment (DavidKorczynski) [reply experimental/c-cpp/runner.py:259]: For `check_call` here it's needed -- a list doesn't do the job. That said, I'm not 100% sure about when subprocess wants a list/string in general.

File: experimental/c-cpp/runner.py [hunk 2]
```diff
@@ -251,6 +251,15 @@ def run_on_targets(target,
               build_heuristics=build_heuristics,
               generator_heuristics=generator_heuristics)
 
+  # Cleanup the OSS-Fuzz docker image
+  clean_up_cmd = [
+      'docker', 'image', 'rm', f'gcr.io/oss-fuzz/{worker_project_name}'
```
Comment (oliverchang) [inline experimental/c-cpp/runner.py:256]: hmm, I thought we were always running `docker run` with `--rm` which should do this cleanup automatically on exit. Is that not the case?

Comment (DavidKorczynski) [reply experimental/c-cpp/runner.py:256]: We build an oss-fuzz project per project that is being analysed. It's the image build from the following that we need to clean up: https://github.com/google/oss-fuzz-gen/blob/4e3798ef7a6af994d7e09094cd399876e701d1a2/experimental/c-cpp/runner.py#L67-L72

i.e. after the execution there will be in the shared folders:
```
build/out/temp-project-0
build/out/temp-project-1
...
```
each corresponding to auto-generation of a given repository

Comment (DonggeLiu) [reply experimental/c-cpp/runner.py:256]: I thought `docker image rm` removes the image while `docker run --rm` removes the container?

**Pull request merged**